tests-behat/bootstrap/Context.php

```diff
@@ -93,13 +94,11 @@ protected function disableDebounce(): void
     }
 
     /**
-     * Sleep for a certain time in ms.
+     * Wait for a certain time in ms.
      *
-     * @Then I sleep :arg1 ms
-     *
-     * @param $arg1
+     * @Then I wait :arg1 ms
      */
-    public function iSleep($arg1)
+    public function iWait($arg1)
```

Comment:
We sleep with `sleep` command / fied time, not wait for something.

Reply:
Has a use I wait, not sleep and the BDD you should describe what the user does.
Plus look at the method name for mink `$this->getSession()->wait($arg1);` It say wait not sleep.

Reply:
User sleeps. It does nothing. When user wait, he can do something - like browse Facebook.

Why is code `sleep` is not called `wait`?

Reply:
I do not agree with you on this one. User wait for something to finish. Like an animation or page loading.

Reply:
Again:

- SLEEP - you freeze in time and wake up after some time
- WAIT - you do something in meanwhile, like checking if finish, browse Facebook ;-), etc.

And here - nothing is checked, we simply dummy wait, thus SLEEP.

Makes sense?

Reply:
Not from a user standpoint, it does not. BDD is about user behaviour...

Reply:
Do you agree with SLEEP/WAIT definition, ie. if conditional or dummy? Then what the user does here?

Reply:
he is waiting...

Reply:
for buzzer/delay?